tests/suites/test_suite_aes.function

```diff
@@ -289,6 +289,58 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_CIPHER_MODE_OFB */
+void aes_encrypt_ofb( int fragment_size, char *hex_key_string,
+                        char *hex_iv_string, char *hex_src_string,
```

Comment:
Consider aligning parameters, for style.

Reply:
Done.